docs/source/research/map-jupyterlab-frontend-architecture/README.md

```diff
@@ -0,0 +1,1117 @@
+# Mapping JupyterLab's user interface to source code
+
+Author: Gabriel Fouasnon
+
+Date: January 18, 2022
```

Comment:
It would make more sense to have this as `last updated`

Reply:
Updated!